src/main/java/com/conveyal/datatools/manager/controllers/api/FeedSourceController.java

```diff
@@ -107,11 +107,12 @@ private static FeedSource createFeedSource(Request req, Response res) throws IOE
             }
             // Notify project subscribers of new feed source creation.
             Project parentProject = Persistence.projects.getById(newFeedSource.projectId);
-            NotifyUsersForSubscriptionJob.createNotification(
+            NotifyUsersForSubscriptionJob notifyUsersForSubscriptionJob = new NotifyUsersForSubscriptionJob(
                 "project-updated",
                 newFeedSource.projectId,
                 String.format("New feed %s created in project %s.", newFeedSource.name, parentProject.name)
             );
+            notifyUsersForSubscriptionJob.run();
```

Comment:
The completion of this notification job could happen independently of this request. Also, if the notify job fails, I believe we wouldn't want to fail the request. Therefore this should be added to the datatools light executor (which is a single thread pool for jobs that shouldn't take much processing time).

So this line would become:

```java
DataManager.lightExecutor.execute(notifyUsersForSubscriptionJob);
```

Reply:
Scratch that, I think the previous way should be used. Please revert due to reasons list in https://github.com/ibi-group/datatools-server/pull/348#discussion_r512341045.

Reply:
Updated.